### DISCUSSION TITLE
v0.5.0

### DISCUSSION THREAD

**Discussion by robbievanleeuwen** [robbievanleeuwen/concrete-properties#75] [Announcements]
## :tada: New Features :tada:
- 🇳🇿 NZS3101 Design Code & NZSEE C5 Assessment Guidelines (#40), big thanks to @Agent6-6-6. See more in the [docs](https://robbievanleeuwen.github.io/concrete-properties/user_guide/design_codes/nzs3101.html) and in [this example](https://robbievanleeuwen.github.io/concrete-properties/notebooks/nzs3101.html)!
- 🏹 Prestressed Concrete Analysis (#73). See more in the [docs](https://robbievanleeuwen.github.io/concrete-properties/user_guide/prestressed_analysis.html) and in [this example](https://robbievanleeuwen.github.io/concrete-properties/notebooks/prestressed_section.html)!
- :chart_with_upwards_trend: Modified Mander Non-Linear Profile (Confined & Unconfined Concrete) (#40), thanks to @Agent6-6-6. See more in the [docs](https://robbievanleeuwen.github.io/concrete-properties/user_guide/materials.html#modified-mander-non-linear-unconfined-confined-concrete-service-profile)!

## ❌ Breaking Changes ❌ 
- Axial loads in ``AS3600.ultimate_bending_capacity()`` and ``AS3600.biaxial_bending_diagram()`` are now defined through the ``n_design`` argument (previously ``n``) (#72).

**Full changelog:** [v0.4.1...v0.5.0](https://github.com/robbievanleeuwen/concrete-properties/compare/v0.4.1...v0.5.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/robbievanleeuwen/concrete-properties/releases/tag/v0.5.0'>v0.5.0</a>.</em>

**Comment by connorferster** [comment]
@Agent6-6-6 The NZ design code example is awesome! Love it!

**Comment by Lomarandil** [comment]
Way to go guys! Well done.

Merry Christmas, and I'll look forward to digging through this with an eye
toward adapting for ACI

Lo
